### ISSUE TITLE
Sounds Tagging

### ISSUE THREAD

**Issue by rohrlach** [upstage-org/limelight#93]
Users can tag Sound resources using the same tagging infrastructure we already have

**Comment by helenvarley** [comment]
from https://limelight.upstage.org.nz/sounds, i clicked on the "Edit" button for an audio item, it only gives me the option to change the name or browse to a new file. there is no option to add or delete tags.
from the same page, if i click on the name of the audio item, then i get the opportunity to add or delete tags. this seems quite illogical - surely both are "edit" screens, & it should be possible to access the same functions??
adding & deleting tags appears to work, although when i first tried to add a tag to the "scream", nothing happened. i tried again and it worked.

**Comment by rohrlach** [comment]
That's true - I should add tags to the edit screen too.. or I might consolidate them

**Comment by rohrlach** [comment]
@helenvarley I've made the change as per your feedback (And also applied it to Avatars and Stages).  Should deploy in about an hour.

**Comment by helenvarley** [comment]
it's still not the same screen, clicking on the edit button or on the media item name. from the name you can add or delete tags, & listen to the file. from the edit button, you can do that and edit the name & replace the file. the edit button is still there when you click on the name, but why not go directly to edit? what's the purpose of the intermediary screen where you can only add/delete tags?

**Comment by rohrlach** [comment]
Ahh... I see what you're saying.

It's a convention. You can of course, raise an issue to remove these.

**Comment by vickis** [comment]
ideally the tags would be added initially on upload (as per good meta data processes :)

**Comment by rohrlach** [comment]
@vickis - agreed. But that's actually a non trivial task so can we have a separate issue for adding tags at creation time?
